lambdas/Makefile

```diff
@@ -0,0 +1,25 @@
+ROOT = $(shell git rev-parse --show-toplevel)
+
+ifneq ($(ROOT), $(shell pwd))
+	include $(ROOT)/shared.Makefile
+endif
+
+
+.docker/_lambda_deps: .docker/python3.6_ci
```

Comment:
I think you want this to be `$(ROOT)/.docker/_lambda_deps`, so it always goes to `.docker` in the root of the repo, regardless of whether you run `make` from root or from within this directory.

Also, `.docker/terraform_ci` and `.docker/python3.6_ci` and maybe `uptodate-git` need to move into shared.Makefile.

Reply:
👍  but boing to leave `uptodate-git` as it doesn't quite feel like ti fits the pattern of waht we're putting in the `shared.Makefile` at the moment.